assets/css/v2/dup/departures/overnight.scss

```diff
@@ -0,0 +1,44 @@
+.overnight-departures__container {
+  height: 832px;
+  .overnight-departures__image {
+    width: 1920px;
+
+    &--full {
+      height: 472px;
+    }
+
+    &--partial {
+      height: 314px;
+    }
+  }
+
+  .overnight-departures__text-container {
+    margin-left: 71px;
+
+    .overnight-departures__route-pill-container {
+      padding-top: 33px;
+
+      .route-pill {
+        display: inline-block;
+        height: 144px;
+        width: 279px;
+        border-radius: 100px;
+        margin: 0 40px 0 0;
+      }
+    }
+
+    .overnight-departures__text {
+      width: 1182px;
+      font-family: Inter, sans-serif;
+      font-weight: 600;
+      font-size: 138px;
+      line-height: 148px;
```

Comment:
This is actually an `overnight-section`, so it uses `FreeText` instead of that file. The `line-height` was not correct for `overnight-section`s so I got that fixed. Also gave `departure-section`s a static height of `416px` and reduced the text box width. 

With the change to section height, the divider was not always rendering in the right place under some section types. A more reliable way to position it is to flip `&:not(:last-child):after` to `&:not(:first-child):before`. So that's why that change in in the most recent commit.